aiohue/lights.py

```diff
@@ -61,6 +61,11 @@ def swversion(self):
         """Software version of the light."""
         return self.raw['swversion']
 
+    @property
+    def swupdateavailable(self):
+        """Software update available for the light."""
+        return self.raw.get('swupdate', {}).get('state', 'not updatable')
```

Comment:
I don't think that we should fill in `not updatable` when there is no value. Instead just let it return `None`

Reply:
Alright, will change it